projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/create.py

```diff
@@ -40,7 +46,7 @@ def create_job(
             self.__create_cloud_job(team, name)
         if local:
             job_path = self.__get_job_path(path, name)
-            self.__create_local_job(team, name, job_path)
+            self.__create_local_job(team, name, job_path, sample_job_directory)
```

Comment:
May consider reusing `__get_job_path()` validation in a way, similar to L48. 
For example, `job_path` arg is already validated to be an actual path.